### PR TITLE
Added mish activation function and test for PaddlePaddle frontend

### LINKED ISSUE #16249: mish

### PR DESCRIPTION
**Pull request by sachelsout**
Close #16249

### CODE REVIEW

File: ivy/functional/frontends/paddle/nn/functional/activation.py [hunk 1]
```diff
@@ -87,3 +87,9 @@ def softshrink(
     up = ivy.where(ivy.greater(x, threshold), ivy.subtract(x, threshold), 0)
     add = ivy.add(low, up)
     return ivy.astype(add, x.dtype)
+
+
+@with_supported_dtypes({"2.4.2 and below": ("float32", "float64")}, "paddle")
```
[the remark this answers is not in the document]
Comment (sachelsout) [reply ivy/functional/frontends/paddle/nn/functional/activation.py]: > Can we justify the selection of float32 and float64 only? Is there any reference to that?

Hi, rechecked again and there's no reference for these dtypes selection. 
May I know what we can do, in such situations, where dtypes are not mentioned in the official docs ?

[the remark this answers is not in the document]
Comment (ZiadAmerr) [reply ivy/functional/frontends/paddle/nn/functional/activation.py]: Try to run the code yourself, if it raises an error, then it shouldn't be supported. If it doesn't raise an error but outputs inconsistent results compared to other frameworks, you should flag it.

[the remark this answers is not in the document]
Comment (sachelsout) [reply ivy/functional/frontends/paddle/nn/functional/activation.py]: I tried running the code for different dtypes. Only float32 and float64 are supported for mish. For other dtypes, code is returning an error. 
image
It's also mentioned in the paddle backend for mish activation function.

[the remark this answers is not in the document]
Comment (sachelsout) [reply ivy/functional/frontends/paddle/nn/functional/activation.py]: Hello @ZiadAmerr , could you please check this out.

**Comment by ZiadAmerr** [comment]
Hi @sachelsout, are the tests passing locally?

**Comment by sachelsout** [comment]
> Hi @sachelsout, are the tests passing locally?

Hi @ZiadAmerr , all the tests are passing locally, except for jax framework. 
image
But I tried running other functions tests, and all of them are failing for jax. I use codespaces and its running on latest ivy image. I think it might be due to recent changes in the jax related libraries' versions. Just my guess.

**Comment by ZiadAmerr** [comment]
Are the tests passing for your function? Try running the tests for your testing function only.

**Comment by sachelsout** [comment]
> Are the tests passing for your function? Try running the tests for your testing function only.

As I mentioned earlier, the tests are passing for my function, except the test for jax framework.

**Comment by ZiadAmerr** [comment]
> > Are the tests passing for your function? Try running the tests for your testing function only.
> 
> 
> 
> As I mentioned earlier, the tests are passing for my function, except the test for jax framework.

I thought you meant other functions. Well then you should debug that and see why the error is rising in JAX and try to fix it. :)

**Comment by sachelsout** [comment]
> > > Are the tests passing for your function? Try running the tests for your testing function only.
> > 
> > 
> > As I mentioned earlier, the tests are passing for my function, except the test for jax framework.
> 
> I thought you meant other functions. Well then you should debug that and see why the error is rising in JAX and try to fix it. :)

The error is probably due to codespace usage. People with linux+docker setup aren't facing this issue of jax. I read the conversations regarding this issue on Ivy's Discord channels. 
My code snippet for this function is correct. This error is outside the scope of this function. Mostly it has to do with the jax backend.

**Comment by ZiadAmerr** [comment]
Can you provide a screenshot of the test error

**Comment by sachelsout** [comment]
> Can you provide a screenshot of the test error

I've attached it in the above discussion. It's the same error for all the functions.

**Comment by ZiadAmerr** [comment]
Hey @sachelsout, I think it is a problem with a new JAX version, as per our discussion on Discord, we are still working on resolving this problem, when we do we can rerun the tests and make sure that everything is working fine :)

**Comment by sachelsout** [comment]
> Hey @sachelsout, I think it is a problem with a new JAX version, as per our discussion on Discord, we are still working on resolving this problem, when we do we can rerun the tests and make sure that everything is working fine :)

Sounds good @ZiadAmerr 😊 thanks for the update !!

**Comment by ZiadAmerr** [comment]
Hey @sachelsout, I think the problem has been fixed! Can you try to run the tests locally with the latest changes from master, if the tests are passing locally for you, I will do a couple reruns on my end and we can proceed with the merge :)

**Comment by sachelsout** [comment]
> Hey @sachelsout, I think the problem has been fixed! Can you try to run the tests locally with the latest changes from master, if the tests are passing locally for you, I will do a couple reruns on my end and we can proceed with the merge :)

Hi @ZiadAmerr , I ran the tests locally and all are passing 😃 !!

image

**Comment by ZiadAmerr** [comment]
Amazing, they passed on my end too! Thanks a lot @sachelsout :)

**Review by ZiadAmerr — APPROVED** [review]

**Comment by sachelsout** [comment]
> Amazing, they passed on my end too! Thanks a lot @sachelsout :)

Thank you @ZiadAmerr for reviewing and merging the code ! 😊

**Comment by ZiadAmerr** [comment]
You're welcome @sachelsout! Keep up the amazing work. :)

**Pull request merged**